### DISCUSSION TITLE
Is it possible to use key fallback on context?

### DISCUSSION THREAD

**Discussion by dutbu2718** [i18next/i18next#1818] [Q&A] [answered]
Hi, 
I have been playing around with context lately and is wondering if it is possible to use key fallback on context? 

Say if I have: 
```
{
      "friend": "A friend",
      "friend_male": "A boyfriend",
      "friend_female": No value
}
```

And want: 
```
i18next.t('friend'); // -> "A friend"
i18next.t('friend', { context: 'male' }); // -> "A boyfriend"
i18next.t('friend', { context: 'female' }); // -> " friend_female "  but instead I get  "A friend"

```
My config is:
```
  i18n: {
    defaultLocale: 'en',
    locales: ['en'],
  },
```

**Comment by jamuhl** [comment, marked as answer]
Context fallbacks to the value without `_[givenContext]` -> therefore your sample fallbacks to the "neutral" value `friend` -> remove that and it would fallback to key.

**Reply by dutbu2718** [reply]
It works with `i18next.t('friend_female'); // -> " friend_female"`. Thanks 😄.
